blog/2025-03-31-a-more-accessible-captcha-widget/index.md

```diff
@@ -0,0 +1,20 @@
+---
+slug: a-more-accessible-captcha-widget
+title: "A more accessible captcha widget"
+authors: [agreenberg]
+tags: [captcha, accessibility, announcement]
+---
+
+A couple of weeks ago, while browsing open support tickets, I noticed a customer report that the Friendly Captcha widget was not WCAG-compliant. In other words, something about the widget fell outside the set of recommendations provided by the Web Content Accessibility Guidelines.
+
+Removing barriers to accessing the web is a primary value of Friendly Captcha, so it wouldn't do to have a product that was not as accessible as possible. As luck would have it, right around the time of this customer report, we launched an in-house [website accessibility checker](https://accessibilitycheck.friendlycaptcha.com/). It allows anyone to input a website URL and receive a report with an accessibility score and list of remediations to achieve compliancy with common accessibility standards.
```

Comment:
```suggestion
Removing barriers to accessing the web is a primary value of Friendly Captcha, so it wouldn't do to have a product that was not as accessible as possible.

As luck would have it, right around the time of this customer report, we launched an in-house [website accessibility checker](https://accessibilitycheck.friendlycaptcha.com/). It allows anyone to input a website URL and receive a report with an accessibility score and a list of remediations to achieve compliancy with common accessibility standards such as **WCAG 2.1** and **EN 301 549**.
```

(Added a linebreak, added a missing `a` ("a list of"), and named the two main a11y standards.

Reply:
"It wouldn't do" is an expression I fear a non-native speaker will have never heard before